whipper/result/result.py

```diff
@@ -153,7 +153,7 @@ def getLoggers():
     """
     Get all logger plugins with entry point 'whipper.logger'.
 
-    @rtype: dict of C{str} -> C{Logger}
+    :rtype: dict of str -> :any:`Logger`
```

Comment:
```suggestion
    :rtype: dict of :class:`str` -> :class:`Logger`
```
Maybe?

Reply:
I've found one instance where L{} was used instead of C{}. If you think it's better this way, I'll change it (don't mind).

Reply:
I don’t really care for `:any:Logger` vs. `:class:Logger` (in the context of this PR), but `str` at least should get the `:class:`. :)

Reply:
Done!